kickstart/virt-install.sh

```diff
@@ -7,7 +7,7 @@ sudo virt-install \
   --os-variant fedora-unknown \
   --boot uefi \
   --vcpus 4 --ram 16384 \
-  --disk=/var/lib/libvirt/images/playtron-os.img,bus=virtio,format=raw,size=9 \
+  --disk=/var/lib/libvirt/images/playtron-os.img,bus=virtio,format=raw,size=12 \
```

Comment:
Now that we're using the `fedora-bootc` image, can this size be reduced now? Perhaps even lower than the original 9 GiB we were using before?